src/devices/Card/CreditCard/WorldCountryCurrency.cs

```diff
@@ -0,0 +1,284 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+
+namespace Iot.Device.Card.CreditCardProcessing
+{
+    public class WorldCountryCurrency
+    {
+        /// <summary>
+        /// List of countries, currency names, currency codes, currency code number and country code number
+        /// Use this list to display a country name when extracting the currency or country code names from the card
+        /// </summary>
+        public static List<CountryCurrency> CountryCurrencies = new List<CountryCurrency>()
```

Comment:
@tarekgh do you perhaps know if we have this data somewhere already perhaps?

Reply:
I searched for it but didn't find. If we have anywhere, happy to reuse.

Reply:
We have RegionInfo which include some of the information populated here. 

I believe we can get the rest of the information from the underlying OS. 

I don't like hardcoding the information here because this will require us to update the library every time something change in this table. especially currency information is changing all the time. It will be good if we can try populate the information dynamically from other source.


---
In reply to: [308507153](https://github.com/dotnet/iot/pull/598#discussion_r308507153) [](ancestors = 308507153)

Reply:
> I don't like hardcoding the information here because this will require us to update the library every time something change in this table. especially currency information is changing all the time. It will be good if we can try populate the information dynamically from other source.

I understand, so I will just remove it. It's mainly to display nicely the information from the card. Used that in a project as RegionInfo does not allow to get a country name from an ISO Code currency code for example. I may then open an issue to be able to get a region based on something else than the culture id or culture string.

Reply:
Filled an Issue here: https://github.com/microsoft/referencesource/issues/114#issue-475096082